pkg/occlient/occlient.go

```diff
@@ -163,6 +165,43 @@ func New(connectionCheck bool) (*Client, error) {
 	return &client, nil
 }
 
+func (c *Client) Login(username, password, token string) error {
```

Comment:
it might be good idea to put login related commands to separate file, as this file is starting to get big, and it kind of make sense to start breaking it into logical pieces